clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1119,6 +1119,9 @@ public boolean shouldBlock() {
      * <p>
      * This is a synchronous commits and will block until either the commit succeeds or an unrecoverable error is
      * encountered (in which case it is thrown to the caller).
+     * <p>
```

Comment:
I notice that `commitAsync` doesn't say anything about ordering of callbacks. Should we say something there too?